ui/selectors/permissions.js

```diff
@@ -374,6 +374,47 @@ export function getOrderedConnectedAccountsForActiveTab(state) {
     );
 }
 
+export function createGetOrderedConnectedAccountsForActiveTabMemo() {
+  return createDeepEqualSelector(
```

Comment:
This is a larger win, since this selector (above) was constantly returning a new array.
So almost every time the selector fires from a redux state change, it would cause a component render.